### PR TITLE
Set up of links between apps and grants

### PR DESCRIPTION
**Pull request by thelifeandtimes**

### CODE REVIEW

File: content/grants/2019/bitcoin-node-json-rpc-api-part-1.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ assignee = ["yosoyubik"]
 id = "649469235"
 completed = true
 link = ""
+link = "~pindet-timmut/urbitcoiners"
```
Comment (matildepark) [inline content/grants/2019/bitcoin-node-json-rpc-api-part-1.md:16]: It's failing because `link = ` is written twice on all these.

Comment (thelifeandtimes) [reply content/grants/2019/bitcoin-node-json-rpc-api-part-1.md:16]: ah, thanks.

**Comment by thelifeandtimes** [comment]
@matildepark, i fixed those metadata issues but now is throwing a more complex set of errors.

**Comment by thelifeandtimes** [comment]
@matildepark thanks for the fix. Can we exclude from the app pages any grants where `canceled = true`? those are excluded on the grants main page and I think they would make sense to exclude on app pages as well.

**Comment by thelifeandtimes** [comment]
Also, would be good to sort the grants on the app pages with most recent at the top?

**Comment by matildepark** [comment]
I can't see an example of any pages with these cases, but that should do it.

**Comment by thelifeandtimes** [comment]
the bitcoin app is the one where I noticed both these things. This looks to have fixed it; thanks!

**Pull request merged**